pkg/config/assets/configs/required_config.json

```diff
@@ -1 +1,5 @@
-{}
+{
+  "username":"a-username-is-required",
+  "password":"a-password-is-required",
```

Comment:
These are required in the sense that if they are not provided, the server cannot be contacted and you cannot do anything with it.
It will run. It will be useless.

Reply:
issue in #43